src/sentry/api/serializers/models/metrics_extraction_rules.py

```diff
@@ -1,18 +1,39 @@
+from typing import Any
+
 from sentry.api.serializers import Serializer
+from sentry.sentry_metrics.models import SpanAttributeExtractionRuleConfig
 
 
 class MetricsExtractionRuleSerializer(Serializer):
     def __init__(self, *args, **kwargs):
         Serializer.__init__(self, *args, **kwargs)
 
-    def get_attrs(self, item_list, user):
-        return {item: item.__dict__ for item in item_list}
-
     def serialize(self, obj, attrs, user):
         return {
             "spanAttribute": attrs.get("span_attribute"),
             "type": attrs.get("type"),
             "unit": attrs.get("unit"),
             "tags": list(attrs.get("tags") or set()),
             "conditions": list(attrs.get("conditions") or set()),
+            "id": attrs.get("id"),
         }
+
+
+class SpanAttributeExtractionRuleConfigSerializer(Serializer):
+    def __init__(self, *args, **kwargs):
+        Serializer.__init__(self, *args, **kwargs)
```

Comment:
```suggestion
        super().__init__(*args, **kwargs)
```

You should be able to use super() here.